modules/api/src/main/java/org/apache/ignite/table/manager/IgniteTables.java

```diff
@@ -58,41 +59,12 @@
      */
     CompletableFuture<Table> createTableAsync(String name, Consumer<TableChange> tableInitChange);
```

Comment:
Here and below. Let's actualize javadoc for all create/alter/drop methods in a following way:
- For async - specify that the result future will be completed with TableAlreadyExistsException/TableNotFoundException.
- For sync - add throws clause with corresponding exception.

Reply:
Done.